worker/jobTypes/githubJob.js

```diff
@@ -40,6 +42,56 @@ class GitHubJobClass {
         }
         return false;
     }
+    
+    async writeEnvFile(isProdDeployJob){
+      let pathPrefix;
+
+      if(isProdDeployJob){
+        //download published branches file to check if repo is versioned 
+        const repoObject = {
+          repoOwner: this.currentJob.payload.repoOwner, repoName: this.currentJob.payload.repoName,
+        };
+        const repoContent = await workerUtils.getRepoPublishedBranches(repoObject)
+        //versioned repo
+        if(repoContent && repoContent.content.version.active.length > 1){
+          pathPrefix = `${this.currentJob.payload.repoName.replace('docs-','')}/docsworker-xlarge/${this.currentJob.payload.branchName}`; 
```

Comment:
It seems like sometimes we use the current user and sometimes we hardcode the user. We should not hardcode the user -- when we aren't using current user we should be using system user.

Reply:
this is also a clean up issue! This will not be the case after this afternoon